security-council-rehearsals/2024-01-24-pause-mainnet/README.md

```diff
@@ -0,0 +1,215 @@
+# OP Mainnet Pause
+
+## Objective
+
+This rehearsal is intended to generate pre-signed transactions
+to pause OP Mainnet.
+
+The call that will be executed by the Safe contract is defined in a
+json file. This will be the standard approach for all transactions.
+
+Note that no onchain actions will be taking place during this
+signing. You won’t be submitting a transaction and your address
+doesn’t even need to be funded. These are offchain signatures produced
+with your wallet which will be collected by a Facilitator will execute
+the contract, submitting all signatures for its execution.
+
+Execution can be finalized by anyone once a threshold of signatures
+are collected, so a Facilitator will do the final execution for
+convenience.
+
+## Approving the transaction
+
+### 1. Install tools
+
+```
+just install
+```
+
+### 2. Setup Ledger
+
+Your Ledger needs to be connected and unlocked. The Ethereum
+application needs to be opened on Ledger with the message “Application
+is ready”.
+
+Check your address is correct by running the following command
+
+```
+eip712sign --address --hd-paths "m/44'/60'/0'/0/0" --ledger
+```
+
+You should see the output similar to the following:
+
+```
+Signer: 0x8C835568fE7Eea01B6DCD79214aB5BCe5E1759B0
+``` 
+
+If this is not the address you are expecting,
+you can change the `hd-paths` parameter, i.e. `--hd-paths "m/44'/60'/1'/0/0"`.
+
+### 3. Sign the transactions
+
+We currently have 5 transactions to be signed inside the `tx` folder:
+- `draft-86.json`
+- `draft-unpause-87.json`
```

Comment:
what is the rationale for this one unpause transaction?

Reply:
we'll be pausing and unpausing the chain